mr2022-backgroundtaskmessage-notification-release-1pct.toml

```diff
@@ -0,0 +1,199 @@
+
+(
+SELECT
+    events.metrics.uuid.background_update_client_id AS client_id,
+    mozfun.map.get_key(event.extra, 'branch') AS branch,
+    MIN(DATE(events.submission_timestamp)) AS enrollment_date,
+    COUNT(events.submission_timestamp) AS num_enrollment_events
+-- We need to query from the Glean `background_update` table because pre-[Bug
+-- 1794053](https://bugzilla.mozilla.org/show_bug.cgi?id=1794053) (scheduled for
+-- Firefox 109) we don't have the legacy client ID in
+-- `mozdata.firefox_desktop_background_update.events`.
+FROM `moz-fx-data-shared-prod.firefox_desktop_background_update.background_update` events,
+UNNEST(events.events) AS event
+WHERE
+    DATE(submission_timestamp) BETWEEN
+        '{{experiment.start_date_str}}' AND
+        -- Here we can restrict to the last enrollment date range.
+        '{{experiment.last_enrollment_date_str}}'
+    AND event.category = 'nimbus_events'
+    AND event.name = 'enrollment'
+    -- The background update experiment slug is exact.
+    AND mozfun.map.get_key(event.extra, 'experiment') = '{{experiment.normandy_slug}}'
+    -- This should never happen, but belt-and-braces.
+    AND events.metrics.uuid.background_update_client_id IS NOT NULL
+GROUP BY client_id, branch
+)
+
+UNION ALL
+
+(
+SELECT
+    m.metrics.uuid.background_update_client_id AS client_id,
+    experiment.value.branch AS branch,
+    MIN(DATE(submission_timestamp)) AS enrollment_date,
+    -- These aren't discrete events, it makes no sense to count them.
+    1 AS num_enrollment_events
+-- We need to query from the Glean `background_update` table because pre-[Bug
+-- 1794053](https://bugzilla.mozilla.org/show_bug.cgi?id=1794053) (scheduled for
+-- Firefox 109) we don't have the legacy client ID in
+-- `mozdata.firefox_desktop_background_update.events`.
+FROM `mozdata.firefox_desktop_background_update.background_update` AS m
+CROSS JOIN
+    UNNEST(ping_info.experiments) AS experiment
+WHERE
+    -- Background update telemetry can be delayed, so we accept enrollment
+    -- _submission_ dates during the entire experiment.  It's safer to compare
+    -- submission dates generated server-side than internal ping dates generated
+    -- client-side.
+    DATE(submission_timestamp) BETWEEN
+        '{{experiment.start_date_str}}' AND
+        '{{experiment.end_date_str}}'
+    -- The background update experiment slug is exact.
+    AND experiment.key = '{{experiment.normandy_slug}}'
+GROUP BY client_id, branch
+)
+
+UNION ALL
+
+(
+SELECT
+    client_id AS client_id,
+    -- Post [Bug 1804988](https://bugzilla.mozilla.org/show_bug.cgi?id=1804988),
+    -- this name looks like 'slug:branch'.  For the
+    -- 'mr2022-backgroundtaskmessage-notification-release-1pct' experiment, this
+    -- name is 'mr2022-backgroundtaskmessage-notification-release'.  In that
+    -- case, the sole branch is 'enabled'.
+    CASE
+        WHEN '{{experiment.normandy_slug}}' = 'mr2022-backgroundtaskmessage-notification-release-1pct' THEN 'enabled'
```

Comment:
What is this line doing? Wouldn't `{{experiment.normandy_slug}}` always be `'mr2022-backgroundtaskmessage-notification-release-1pct'` when this is run?